test/e2e/policies.test-suite/static-pools/static-pools-lib.source.sh

```diff
@@ -0,0 +1,25 @@
+# shellcheck disable=SC2148
+static-pools-relaunch-cri-resmgr() {
+    local webhook_running=0
+    out "# Relaunching cri-resmgr and agent, launch webhook if not already running"
+
+    vm-command-q "kubectl get mutatingwebhookconfiguration/cri-resmgr" >/dev/null 2>&1 && {
```

Comment:
extreme nitpick: since we assume bash, isn't `>& /dev/null` an easier/more readable way to write this ?

Reply:
Thanks, fixed!